front/src/BasicTable.tsx

```diff
@@ -9,42 +9,56 @@ import { useTranslation } from 'react-i18next';
 import { Button } from '@material-ui/core';
 import "./BasicTable.css";
 
-export default function BasicTable() {
-  const [offers, setOffers] = useState<any[]>([]); // TODO fix any
+
+interface OffersTypes {
```

Comment:
just `Offer` plz. No need to use plurals and suffixing with "type", "interface" etc

Reply:
BTW. If I remember correctly, some of these fields are optional. You could examine the sample offers for API and determine, which ones are nullable.